test/random.js

```diff
@@ -44,7 +44,7 @@ client.on('message', message => {
             count++;
             console.log('reached', count, ecount);
           })
-          .catch(m => {
+          .catch(e => {
             console.error(m);
```

Comment:
Seems when you changed it to be `e`, the following line didn't change its variable `console.error(e)`!